src/platform/cannonlake/include/platform/lib/memory.h

```diff
@@ -215,8 +215,12 @@
 #define HP_SRAM_WIN3_SIZE	SRAM_TRACE_SIZE
 
 #define HEAP_HP_BUFFER_BASE		(SRAM_TRACE_BASE + SRAM_TRACE_SIZE)
+#if CONFIG_COMP_KPB
+#define HEAP_HP_BUFFER_SIZE		0x40000
+#else
 #define HEAP_HP_BUFFER_SIZE		0x20000
-#define HEAP_HP_BUFFER_BLOCK_SIZE	0x180
```

Comment:
Why did you change block size?

Reply:
Because, as described in commit message smaller blocks allows more accurate allocations.

Reply:
ok, thanks